### PR TITLE
Log level configuration for 2.0.x

### PR DESCRIPTION
**Pull request by halles**
Added configuration for minimal log level config on 2.0.x

Parameter name is the same as what has been implemented for 3.0.x

Update README accordingly.

### CODE REVIEW

**Comment by phptek** [comment]
OK, I ran a few tests and it works well - thanks! I have added a few comments inline. Once you've fixed those up, can you please squash all your commits into a single one and force-push the feature-branch? Cheers :-)

**Comment by halles** [comment]
@phptek this is failing to build do to a silverstripe cms recipe not being available anymore. This tag seems to be two years old. Would you like me to patch that pipeline?

**Comment by phptek** [comment]
Sure! Just needs to be the minimum version that composer will work with.

**Comment by phptek** [comment]
I'm still not sure under what circumstances one might run this task. Can you explain? If it's still required, it should probably be restricted to running only with a permissioned user.

**Comment by halles** [comment]
@phptek this is not the task 🤔

**Comment by halles** [comment]
@phptek could you rerun this pipeline? It should be ready to merge now :)

**Comment by halles** [comment]
@phptek got confused between PRs.

This PR is both the `log_level` config availability and the task to test successful Sentry connection. Basically is to test that Sentry _is_ working before the app shoot an exception.

File: src/Tasks/TestConnectionTask.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+use Monolog\Logger;
+use Psr\Log\LoggerInterface;
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\Dev\BuildTask;
+
+class TestConnectionTask extends BuildTask
+{
+    protected $title = 'Test Sentry Configuration';
+
+    protected $description = 'Captures message for all levels available';
```
[the remark this answers is not in the document]
Comment (G-Rath) [reply src/Tasks/TestConnectionTask.php]: FYI a single one won't cover both for tooling & IDEs ;)

**Pull request merged**